src/_client.ts

```diff
@@ -123,21 +130,26 @@ export class ChalkClient<TFeatureMap = Record<string, ChalkScalar>>
         opts?.activeEnvironment ??
         process.env._CHALK_ACTIVE_ENVIRONMENT ??
         undefined,
-      clientSecret: valueWithEnvFallback(
-        "clientSecret",
-        opts?.clientSecret,
-        "_CHALK_CLIENT_SECRET"
-      ),
       apiServer: resolvedApiServer,
-      queryServer,
       clientId: valueWithEnvFallback(
         "clientId",
         opts?.clientId,
         "_CHALK_CLIENT_ID"
       ),
+      clientSecret: valueWithEnvFallback(
+        "clientSecret",
+        opts?.clientSecret,
+        "_CHALK_CLIENT_SECRET"
+      ),
+      queryServer,
+      timestampFormat: opts?.timestampFormat ?? TimestampFormat.ISO_8601,
     };
 
-    this.http = new ChalkHTTPService(opts?.fetch, opts?.fetchHeaders, opts?.defaultTimeout);
+    this.http = new ChalkHTTPService(
```

Comment:
Our formatter settings might be fighting here 😄

Reply:
😅 I had prettier on autoformat but there is not a prettierrc yet, I disabled it for now